src/App.tsx

```diff
@@ -7,8 +7,11 @@ import {
 import './index.css';
 import { About, FAQ, Home } from './components/pages';
 import { Layout } from './components/layout';
+import useArNSContract from './hooks/useArNSContract/useArNSContract';
```

Comment:
nit: let's create an index.tx in `hooks` - we'll have a lot

Reply:
:eyes: